README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
I can't remember exactly what the problem was, but you get kafka errors on `./manage.py run_ptop --all` (we could change the startup instructions to `--pillow-key=core`), and on `./manage.py ptop_es_manage --flip_all_aliases`.

Trying it out now, neither of those actually seems blocking (there are errors in the traceback, but the process completes), but that might also be because my local setup is already up-to-date.  The `BlobDeletionPillow` is blocking, however, and has to be commented out.